### PR TITLE
Add AE breaking change

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #17096

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/3.1-5.0?branch=pr-en-us-18049#winforms-methods-now-throw-argumentexception).

### CODE REVIEW

**Review by RussKie — APPROVED** [review]
👍

**Comment by RussKie** [comment]
The docs for `System.Windows.Forms.TabControl.GetToolTipText(System.Object)` be updated as well. Is this something you do, or we need to do it?

**Comment by gewarren** [comment]
> The docs for `System.Windows.Forms.TabControl.GetToolTipText(System.Object)` be updated as well. Is this something you do, or we need to do it?

Typically, the engineering team updates the API docs. We can help with PR reviews though.

File: includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Previously, passing arguments of an unexpected or incorrect type to certain Windows Forms methods would result in an indeterminate state. Starting in .NET 5.0, these methods now throw an <xref:System.ArgumentException> when passed invalid arguments.
+
+Throwing an <xref:System.ArgumentException> conforms to the behavior of the .NET runtime. It also improves the debugging experience by clearly communicating which argument is invalid.
+
+The following table lists the affected methods and parameters:
+
+| Method | Parameter name | Condition | Version added |
+|-|-|-|
+| <xref:System.Windows.Forms.TabControl.GetToolTipText(System.Object)?displayProperty=fullName> | `item` | Argument is not of type <xref:System.Windows.Forms.TabPage>. | 5.0 Preview 1 |
```
Comment (tdykstra) [inline includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md:15]: There's only one item in the table, but all the plurals in the preceding text lead you to expect more than that. Do you expect to add more later?

**Review by tdykstra — APPROVED** [review]
Looks good but I had one question.

File: includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md (hunk 1 above)
Comment (gewarren) [reply includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md:15]: Yes, there's one for preview 5 that I know about but don't want to document yet. Do you think it's okay as-is?

Comment (tdykstra) [reply includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md:15]: In that case, yes.

**Comment by gewarren** [comment]
Thanks for the reviews!

File: includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md (hunk 1 above)
Comment (RussKie) [reply includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md:15]: I think we can expect more to come as we go through the codebase

**Pull request merged**